### PR TITLE
Names for H-Release naming

### PR DESCRIPTION
**Pull request by JPrause**
[skip ci]

### CODE REVIEW

**Comment by JPrause** [comment]
WIP until:
- Names are okay
- Dates defined to open poll
- talk article with Poll is created

@chessbyte @cybette

**Comment by JPrause** [comment]
Added dates based on past polling dates lining up to a beta release.
Poll created on talk.manageiq.org Nov 06
Poll ends Nov 17
Poll announce Nov 20

Won't be able to finalize this PR until the talk poll is created. Need the URL for: http://talk.manageiq.org/t/voting-on-name-of-h-release/???

**Comment by JPrause** [comment]
@miq-bot add_label documentation

**Comment by JPrause** [comment]
@cybette can you merge this for h-name polling.

**Comment by JPrause** [comment]
@chessbyte this is kind of time sensitive since we opened the poll today.
I think Carol may be travelling,..could you merge this PR.

**Comment by JPrause** [comment]
or @Fryguy 👍

File: site/_posts/2017-10-18-vote-on-naming-h-release.html.md [hunk 1]
```diff
@@ -0,0 +1,27 @@
+---
+title: Voting on Name of H Release
+author: chessbyte
+date: 2017-11-06 09:02:24 UTC
+tags: releases announcements
```
Comment (cybette) [inline site/_posts/2017-10-18-vote-on-naming-h-release.html.md]: the voting posts have so far been tagged with releases and collaboration. there's nothing to announce yet :)

**Comment by cybette** [comment]
Sorry, orchestra prac on mondays. One little fix and I'll merge. Thanks!

**Comment by JPrause** [comment]
@cybette cool,..I'll make that change now. and no worries. 👍

**Pull request merged**